doc/04-Security.md

```diff
@@ -0,0 +1,53 @@
+# Security
+
+Icinga for Kubernetes allows users to show different Kubernetes resources. Users may be restricted to a specific set of 
+resources, by use of **permissions** and **restrictions**.
+
+## Permissions
+
+> **Restricted Permissions:**
+>
+> If a role [limits users](#filters) to a specific set of results, the
+> permissions or refusals of the very same role only apply to these results.
+
+| Name                                    | Allow...                         |
+|-----------------------------------------|----------------------------------|
+| kubernetes/config-maps/show             | to show config maps              |
+| kubernetes/cron-jobs/show               | to show cron jobs                |
+| kubernetes/daemon-sets/show             | to show daemon sets              |
+| kubernetes/deployments/show             | to show deployments              |
+| kubernetes/events/show                  | to show events                   |
+| kubernetes/ingresses/show               | to show ingresses                |
+| kubernetes/jobs/show                    | to show jobs                     |
+| kubernetes/nodes/show                   | to show nodes                    |
+| kubernetes/persistent-volume-claims/show | to show persistent volume claims |
+| kubernetes/persistent-volumes/show      | to show persistent volumes       |
+| kubernetes/pods/show                    | to show pods                     |
+| kubernetes/replica-sets/show            | to show replica sets             |
+| kubernetes/secrets/show                 | to show secrets                  |
+| kubernetes/services/show                | to show services                 |
+| kubernetes/stateful-sets/show           | to show stateful sets            |
+| kubernetes/yaml/show                    | to show yaml                     |
+
+## Restrictions
+
+### Filters
+
+Filters limit users to a specific set of results.
+
+> **Note:**
+>
+> Filters from multiple roles will widen available access.
+
+| Name                        | Description                                                       |
+|-----------------------------|-------------------------------------------------------------------|
+| kubernetes/filter/resources | Restrict access to the Kubernetes resources that match the filter |
+
+`kubernetes/filter/resources` will only allow users to access matching Kubernetes resources. This applies to all 
+resources. 
+
+Allowed columns are namespace and name.
+
+> **Note:**
+>
+> Nodes, namespaces and persistent volumes can only be restricted by name.
```

Comment:
Please document that these resources do not belong to a namespace and therefore only the name is available for filtering.